plugins/catalog-backend-module-incremental-ingestion/README.md

```diff
@@ -175,17 +175,17 @@ import { IncrementalEntityProvider } from '@backstage/plugin-catalog-backend-mod
 
 // This will include your pagination information, let's say our API accepts a `page` parameter.
 // In this case, the cursor will include `page`
-interface MyApiCursor {
+interface TCursor {
```

Comment:
We actually would probably prefer just `Cursor` and `Context` over the prefixed `T`.